src/main/java/io/github/bobocodebreskul/context/registry/SimpleObjectFactory.java

```diff
@@ -0,0 +1,50 @@
+package io.github.bobocodebreskul.context.registry;
+
+import io.github.bobocodebreskul.context.config.BeanDefinition;
+import io.github.bobocodebreskul.context.exception.ClassNotFoundRuntimeException;
+import io.github.bobocodebreskul.context.exception.InstanceCreationException;
+import io.github.bobocodebreskul.context.exception.NoSuchMethodRuntimeException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimpleObjectFactory implements ObjectFactory {
+
+  private final Map<Class<?>, Object> storage = new HashMap<>();
```

Comment:
I think - the key should be beanName(String). It's needed for the case when we implement the possibility of creating several instances of one class with different names.